src/database/datatools/standard_materials.py

```
@@ -0,0 +1,159 @@
+                              density: float,
+                              specific_heat: float,
+                              thermal_absorptance: Optional[float] = None,
+                              solar_absorptance: Optional[float] = None,
+                              visible_absorptance: Optional[float] = None) -> None:
+    
+    conn = sqlite3.connect(db_path)
+    cursor = conn.cursor()
+
+    table_name = "standard_materials"
+
+    sql = f"INSERT INTO {table_name} (name, latitude, longitude, architecture_type, roughness, thickness, conductivity, density, specific_heat, thermal_absorptance, solar_absorptance, visible_absorptance, datetime) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)"
```

Ruff (0.14.14)

[error] Possible SQL injection vector through string-based query construction

(S608)